src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -118,6 +122,18 @@ def send_historical_data_to_seer(alert_rule: AlertRule, project: Project) -> Ale
             },
         )
         raise TimeoutError
+    except Exception:
+        logger.warning(
+            "Unexpected error sending data to Seer",
+            exc_info=True,
+            extra={"rule_id": alert_rule.id, "project_id": project.id},
+        )
+        raise UnexpectedSeerError("Unexpected error from Seer. Cannot store historical data.")
+    else:
+        logger.info(
+            "Data successfully sent to Seer",
+            extra={"rule_id": alert_rule.id, "project_id": project.id},
+        )
```

Comment:
Do we need this success log? If nothing fails you could assume everything passed as expected